plugins/defaultplugins/l3plugin/README.md

```diff
@@ -13,20 +13,21 @@ The configuration must be stored in etcd using the following key:
 
 An example of configuration in json format can be found [here](../../../cmd/vpp-agent-ctl/json/arp.json).
 
-To insert config into etcd in json format [vpp-agent-ctl](../../../cmd/vpp-agent-ctl/main.go) can be used.
-We assume that we want to configure vpp with label `vpp1` and config is stored in the `arp.json` file
+To insert config into etcd in json format [vpp-agent-ctl](../../../cmd/vpp-agent-ctl/vpp-agent-ctl.go) 
```

Comment:
Same here. https://github.com/ligato/vpp-agent/pull/507/files#r175028144

Reply:
I'll find all the other places